app/components/ActionPanel.js

```diff
@@ -92,12 +93,17 @@ type Props = {
   updateActiveTab: Function,
   identities: array,
   isLoadingTransactions: boolean,
-  syncWallet: Function
+  syncWallet: Function,
+  selectedAccountHash: string,
+  selectedParentHash: string
 };
 
 type State = {
   activeTab: string,
```

Comment:
I know its not yours but please move it to `type Props = {` we dont have it on state no more.